modules/core/client/components/Map/osm.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 8,
+  "name": "OpenStreetMap",
+  "metadata": {},
+  "sources": {
+    "osm": {
+      "attribution": "<a href=\"https://www.openstreetmap.org/about/\" target=\"_blank\" rel=\"noopener\">&copy; OpenStreetMap</a>",
+      "mapbox_logo": true,
```

Comment:
We wouldn't _need_  to show the Mapbox logo with this layer, but I've left it here so that it's visible while developing and we know to take into account as if we'd be on production site.

Reply:
> I'm not planning on showing OSM in production anyway.

:+1: [OSM tile policy](https://operations.osmfoundation.org/policies/tiles/) disallows distributing an app which use OSM tiles and other heavy uses.

Reply:
Yup, exactly. :-)